client/src/pages/reports/Form.js

```diff
@@ -928,21 +928,23 @@ const ReportForm = ({
                   widget={<RichTextEditor className="reportTextField" />}
                 />
 
-                <FastField
-                  name="uploadAttachments"
-                  component={FieldHelper.SpecialField}
-                  label={Settings.fields.attachment.shortLabel}
-                  widget={
-                    <UploadAttachment
-                      edit={edit}
-                      saveRelatedObject={saveCallback}
-                      getRelatedObject={() => getRelatedObject(values)}
-                    />
-                  }
-                  onHandleBlur={() => {
-                    setFieldTouched("uploadAttachments", true, false)
-                  }}
-                />
+                {Settings.fields.attachment.featureDisabled && (
```

Comment:
Shouldn't this be `!Settings.fields.attachment.featureDisabled && (…` i.e. the inverse condition?

Reply:
https://github.com/NCI-Agency/anet/tree/AB-894-permission-control-to-upload-attachments I did it, github status have error so its not display here